site/en/docs/privacy-sandbox/faq/testing-faq/index.md

```diff
@@ -0,0 +1,68 @@
+-   **PA win reporting**: SSP to DSP
+    -   What SSP's `reportResult()` will make available to the DSP's `reportWin()`.
+-   **Post-auction reporting**: DSP to SSP. Context: The DSP must capture and delegate post-auction reports to the SSP. If the SSP needs clicks, views, viewability metrics, the DSP must enable them to receive these events.
+    -   What reports the SSP will need.
+    -   The methodology for measuring that event (such as, viewability definition).
+    -   DSP implementation of the `reportEvent()` call to support the SSP requirements.
+    -   SSP implementation of the `registerAdBeacon()` lining up the event naming with what the DSP will trigger in the creative.
+
+**Are the Attribution Reporting API and the Aggregation Service ready and available for testing?**
+
+The Attribution Reporting API will be generally available and ramp-up is already in progress. With that, Attribution Reporting will soon be available for use on 100% of traffic. Note that Attribution Reporting can be used with all ads, not just those served by Protected Audience.
+
+**Are there geographic requirements for testing?**
+
+There are no geographic requirements. It's up to each tester to determine geographic considerations as part of their test.
+
+**How does the CMA guidance align with Chrome-facilitated testing?**
+
+Chrome facilitated testing modes are aligned with the CMA's guidance for quantitative testing of Privacy Sandbox. In the CMA's guidance the treatment group relies on Privacy Sandbox technologies without third-party cookies. Control group 1 uses third-party cookies and not Privacy Sandbox, and Control group 2 uses neither Privacy Sandbox nor third-party cookies. With general availability, the Privacy Sandbox technologies will be available on all Chrome traffic, and the ad tech can choose to use Privacy Sandbox technologies in certain population groups and not in others. Additionally, ad techs can leverage the Mode A Chrome-facilitated testing traffic to coordinate these population groups across multiple parties. 
+
+Starting in Q1 2024, Chrome will deprecate third-party cookies on 1% of traffic and this is referred to as the Mode B Chrome-facilitated testing traffic in Chrome developer documentation. In a small fraction of Mode B Chrome-facilitated testing traffic, Privacy Sandbox technologies will also not be available in addition to third-party cookies being deprecated. Using a combination of all of the above Chrome-facilitated testing modes, ad techs will be able to align their test setups to the testing guidance published by the CMA. (Refer to [Chrome-facilitated testing](/docs/privacy-sandbox/chrome-testing/#:~:text=Mode%20A%3A%20Ad%20techs%20can,portion%20of%20all%20Chrome%20users.) which defines Mode A and Mode B and [CMA Guidance](https://assets.publishing.service.gov.uk/media/649d6a5f45b6a2000c3d455f/20230629_CMA_industry_testing_update_B.pdf) defines Control 1, Control 2, Treatment groups (in point number 11)
```

Comment:
Formatting of the last sentence is a bit off—a bracket opened and not closed, a bracket within a bracket, no full-stop at the end et cetera